src/routes/docs/products/ai/integrations/langchain/+page.markdoc

```diff
@@ -281,55 +281,55 @@ export default class AppwriteService {
         queries.push(Query.cursorAfter(cursor));
       }
 
-      const { documents } = await this.databases.listDocuments(
+      const { rows } = await this.tablesDb.listRows(
         databaseId,
-        collectionId,
+        tableId,
         queries
       );
 
-      if (documents.length === 0) {
+      if (rows.length === 0) {
         break;
       }
 
-      cursor = documents[documents.length - 1].$id;
+      cursor = rows[rows.length - 1].$id;
 
-      cumulative.push(...documents);
+      cumulative.push(...rows);
     } while (cursor);
 
     return cumulative;
   }
 }
 ```
 
-The service provides a method to iterate the documents contained within an entire collection, fetching the limit of 100 documents per request.
+The service provides a method to iterate the rows contained within an entire table, fetching the limit of 100 rows per request.
 
 ```js
 const appwrite = new AppwriteService();
 
-const appwriteDocuments = await appwrite.getAllDocuments(
+const appwriteRows = await appwrite.getAllRows(
   process.env.APPWRITE_DATABASE_ID,
-  process.env.APPWRITE_COLLECTION_ID
+  process.env.APPWRITE_TABLE_ID
 );
 
-const documents = appwriteDocuments.map(
-  (document) =>
-    new Document({
-      metadata: { id: document.$id },
-      pageContent: Object.entries(document)
+const rows = appwriteRows.map(
+  (row) =>
+    new Row({
+      metadata: { id: row.$id },
+      pageContent: Object.entries(row)
         .filter(([key, _]) => !key.startsWith('$'))
         .map(([key, value]) => `${key}: ${value}`)
         .join('\n'),
     })
 );
 
-await PineconeStore.fromDocuments(documents, new OpenAIEmbeddings(), {
+await PineconeStore.fromDocuments(rows, new OpenAIEmbeddings(), {
   pineconeIndex,
   maxConcurrency: 5,
 });
 
 ```
 
-Within our function handler, the service is instantiated and used to create an array of LangChain documents. LangChain documents can then be used with the `PineconeStore.fromDocuments` method to retrieve embeddings from OpenAI and upsert them to your Pinecone index.
+Within our function handler, the service is instantiated and used to create an array of LangChain rows. LangChain rows can then be used with the `PineconeStore.fromDocuments` method to retrieve embeddings from OpenAI and upsert them to your Pinecone index.
```

Comment:
langchain rows might be not needed, no idea what they call it.

Reply:
Will double check